scripts/test_rule.py

```diff
@@ -93,7 +93,7 @@ def validate_single_rule(
                 validated_domains.add(dataset.domain)
                 validated_result = engine.test_validation(
                     rule,
-                    os.path.join(directory, dataset.filename),
+                    os.path.join(directory, dataset.filename).replace(os.sep, '/'),
```

Comment:
This fix works, but it defers this OS file-separator issues. It looks like the actual cause of the issue is here:

https://github.com/cdisc-org/cdisc-rules-engine/blob/3c18a77ca164259d20d70f4710b2b094e4a85a76/cdisc_rules_engine/services/data_services/dummy_data_service.py#L51

Instead, it should be something like: 
```python
dataset_name = os.path.basename(dataset_name)
```